layout/base/nsDisplayList.cpp

```diff
@@ -109,7 +109,7 @@ SpammyLayoutWarningsEnabled()
 #endif
 
 void*
-AnimatedGeometryRoot::operator new(size_t aSize, nsDisplayListBuilder* aBuilder)
+AnimatedGeometryRoot::operator new(size_t aSize, nsDisplayListBuilder* aBuilder) CPP_THROW_NEW
```

Comment:
Note: this is an unrelated change for a bug that I happened to run into while developing this patch. I'm unsure why I ran into it, but in any case I've fixed it upstream in https://bugzilla.mozilla.org/show_bug.cgi?id=1291889, and it shouldn't hurt to merge it here in the meantime.